CreateNotificationActivity/handler.ts

```diff
@@ -170,10 +170,13 @@ export const getCreateNotificationActivityHandler = (
   //
   // * email notifications are enabled in the user profile (isEmailEnabledInProfile)
   // * email notifications are not blocked for the sender service (!isEmailBlockedForService)
+  // * the sender service do NOT require secure channels
   // * a destination email address is configured (maybeEmailFromProfile)
   //
   const maybeEmailNotificationAddress =
-    isEmailEnabledInProfile && !isEmailBlockedForService
+    isEmailEnabledInProfile &&
+    !isEmailBlockedForService &&
+    !senderMetadata.requireSecureChannels
```

Comment:
here I tried to keep the code at minimum to make it very easy to understand, I think it's better to create an intermediate variable in the block above like `const isEmailChannelAllowed = !senderMetadata.requireSecureChannels`

Reply:
Done